tools/docs/config.ts

```diff
@@ -1,11 +1,44 @@
-import table from 'markdown-table';
+import stringify from 'json-stringify-pretty-compact';
 import { getOptions } from '../../lib/config/options';
 import { getCliName } from '../../lib/workers/global/config/parse/cli';
 import { getEnvName } from '../../lib/workers/global/config/parse/env';
 import { readFile, updateFile } from '../utils';
 
 const options = getOptions();
 
+function indent(amount: number, indent = '  '): string {
```

Comment:
i think this is a good candidate for a tagged  template function 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates

Reply:
The outcome is a bit over engineered, as we need to support `${col}` and such.
Interesting topic nonetheless. 
we can revert it of course if needed.